src/Components/BrandImage/BrandImage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import '../../Pages/HomePage/HomePage.css'
+
```

Comment:
changes: I think HomePage.css is not required here

Reply:
the styling comes from homepage.css file
`.brand-img-wrapper {
  width: 8.4rem;
  height: 8.4rem;
}`